target/linux/realtek/files-6.18/drivers/net/phy/rtl83xx-phy.c

```diff
@@ -59,7 +60,9 @@ static const struct firmware rtl838x_8218b_fw;
 
 static inline struct phy_device *get_package_phy(struct phy_device *phydev, int port)
 {
-	return mdiobus_get_phy(phydev->mdio.bus, phydev->shared->base_addr + port);
+	int base_addr = phydev->mdio.addr & ~7;
```

Comment:
Don't like that because we have 4 port PHYs like the RTL8214FC. Please check https://github.com/openwrt/openwrt/pull/21204